pkg/i18n/english.go

```diff
@@ -1477,6 +1480,9 @@ func EnglishTranslationSet() TranslationSet {
 		DisabledMenuItemPrefix:              "Disabled: ",
 		NoCommitSelected:                    "No commit selected",
 		NoCopiedCommits:                     "No copied commits",
+		QuickStartInteractiveRebase:         "Quick start interactive rebase",
+		QuickStartInteractiveRebaseTooltip:  "Start an interactive rebase for all commits from HEAD down to the first merge commit or main branch commit.\nIf you would instead like to start an interactive rebase from the selected commit, press {{.editKey}} instead",
```

Comment:
Nitpick: there are two `instead`s in the last sentence.

Reply:
haha good catch